### PR TITLE
Release 3.7.0

### PR DESCRIPTION
**Pull request by nicoddemus**
🎆

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +41,9 @@ jobs:
     - env: TOXENV=py36-freeze
       python: '3.6'
     - env: TOXENV=py37
-      python: 'nightly'
+      python: 3.7
```
Comment (RonnyPfannschmidt) [inline .travis.yml]: we should probably put quotes around this at some point

Comment (nicoddemus) [reply .travis.yml]: Oh right, classical YAML snafu. I will do it on `master` after we release this. 👍

Comment (nicoddemus) [reply .travis.yml]: Done! 😉

File: CHANGELOG.rst [hunk 2]
```diff
@@ -8,6 +8,70 @@
...
+
+
+- `#3623 <https://github.com/pytest-dev/pytest/issues/3623>`_: Introduce ``pytester.copy_example`` as helper to do acceptance tests against examples from the project.
+
+
+
+Bug Fixes
+---------
+
+- `#2220 <https://github.com/pytest-dev/pytest/issues/2220>`_: Fix a bug where fixtures overriden by direct parameters (for example parametrization) were being instantiated even if they were not being used by a test.
```
Comment (blueyed) [inline CHANGELOG.rst]: s/overriden/overridden/

File: CHANGELOG.rst [hunk 3]
```diff
@@ -8,6 +8,70 @@
...
+
+
+- `#980 <https://github.com/pytest-dev/pytest/issues/980>`_: Fix truncated locals output in verbose mode.
+
+
+
+Improved Documentation
+----------------------
+
+- `#3295 <https://github.com/pytest-dev/pytest/issues/3295>`_: Correct the usage documentation of ``--last-failed-no-failures`` by adding the missing ``--last-failed`` argument in the presented examples, for they are missleading and lead to think that the missing argument is not needed.
```
Comment (blueyed) [inline CHANGELOG.rst]: s/for they are missleading/because they are misleading/

File: .travis.yml (hunk 1 above)
Comment (blueyed) [reply .travis.yml]: What is the reasoning behind this (out of curiosity)?
Is Travis gently handling the float here?

Comment (asottile) [reply .travis.yml]: travis allows both float and string here and yaml is funtimes

File: CHANGELOG.rst (hunk 3 above)
Comment (nicoddemus) [reply CHANGELOG.rst]: Fixed, thanks!

File: CHANGELOG.rst (hunk 2 above)
Comment (nicoddemus) [reply CHANGELOG.rst]: Fixed, thanks!

**Comment by blueyed** [comment]
@nicoddemus 
Thanks for fixing.
Wonder if you would have preferred me editing/pushing this via Github UI myself?

**Comment by nicoddemus** [comment]
> Wonder if you would have preferred me editing/pushing this via Github UI myself?

Sure, feel free to do so in the future. 👍

**Pull request merged**